app/src/main/java/com/example/ana/exampleapp/SendTest.java

```diff
@@ -35,6 +35,8 @@
     DateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
     private final String[] projection = {
             FeedTestContract.FeedEntry.COLUMN_NAME_TIMESTAMP,
+            FeedTestContract.FeedEntry.COLUMN_LATITUDE,
+            FeedTestContract.FeedEntry.COLUMN_LONGITUDE,
```

Comment:
It is better that you add the latitude and longitude at the end of the projection, otherwise you will need to change the indexes in `c.getInt(i)`.

You also need to add the location to the Document which is sent to the MongoDB database. The Document is created in the function `getDoc()`. You need to add after `append("timeWakeUp", c.getInt(17))` this:

`append("latitude", c.getInt(18)),
append("longitude", c.getInt(19));
`

In case that you have added the latitude and longitude to the end of the projection (as they would be the indexes 18 and 19).

After that, if you query this collection in your database (with `db.mobileTests.find().pretty()`), you should see something like this, but with the latitude and longitude before the menstruation:

![image](https://cloud.githubusercontent.com/assets/16052290/16177068/b8a8bd96-3622-11e6-8761-35d211f73b72.png)